entities/src/main/java/com/flipkart/varadhi/auth/ResourceType.java

```diff
@@ -1,6 +1,7 @@
 package com.flipkart.varadhi.auth;
 
 public enum ResourceType {
+    ROOT("root"),
```

Comment:
required for bootstrapping and org create

Reply:
Idea is that when there is no Org defined, we still need a resource node to hold the bootstrap role bindings. Thats why a ROOT is defined which will hold role bindings required for org creation and role assignment. We can also drive super-users by adding role bindings to this node.